### PR TITLE
Sample application to illustrate side-inputs using stream table join

### PR DESCRIPTION
**Pull request by bharathkk**
An example job to illustrate usage of side inputs using a simple stream table join. The table is populated using the side-input stream.

### CODE REVIEW

File: src/main/java/samza/examples/cookbook/data/Profile.java [hunk 1]
```diff
@@ -39,4 +40,7 @@ public Profile(
     this.company = company;
   }
 
+  public Profile(Map<String, Object> jsonObject) {
```
Comment (prateekm) [inline src/main/java/samza/examples/cookbook/data/Profile.java]: Why do this?

File: src/main/java/samza/examples/cookbook/SideInputExample.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+import org.apache.samza.system.kafka.descriptors.KafkaSystemDescriptor;
+import org.apache.samza.table.Table;
+import org.apache.samza.table.descriptors.TableDescriptor;
+import samza.examples.cookbook.data.PageView;
+import samza.examples.cookbook.data.Profile;
+
+
+/**
+ * In this example, we demonstrate stream table join with the source of data for table coming from side inputs.
+ * The side inputs are no different than a normal stream except they act as the source to populate the table for your
```
Comment (prateekm) [inline src/main/java/samza/examples/cookbook/SideInputExample.java]: > "The side inputs are no different than a normal stream except they act as the source to populate the table for your application".

This is not true. We shouldn't be positioning side inputs as an alternative to stream.sendTo(Table). Side Inputs should be documented as an optimization for populating a Local Table / Store with a bounded periodically updated datastore.

Are we adding detailed documentation about side inputs to the website as well?

**Comment by shanthoosh** [comment]
Is this pull-request no longer necessary? Can we close if not needed.

**Comment by mynameborat** [comment]
I lost access to this account due to 2FC issues. 
Will see if I can close it somehow.

**Comment by shanthoosh** [comment]
Since @mynameborat has lost access to his account, I'm closing this PR.

**Pull request closed without merging**